src/tools/rust-analyzer/crates/hir-expand/src/inert_attr_macro.rs

```diff
@@ -665,10 +665,6 @@ pub const INERT_ATTRIBUTES: &[BuiltinAttribute] = &[
     rustc_attr!(TEST, rustc_layout, Normal, template!(List: "field1, field2, ..."), WarnFollowing),
     rustc_attr!(TEST, rustc_abi, Normal, template!(List: "field1, field2, ..."), WarnFollowing),
     rustc_attr!(TEST, rustc_regions, Normal, template!(Word), WarnFollowing),
-    rustc_attr!(
```

Comment:
Doesn't rust-analyzer deal with these attrs themselves? I don't think we typically tweak this lint ourselves from within the rust-lang/rust repo when modifying internal attrs.

Maybe someone like @Veykril would know?

Reply:
It's fine. We auto-generate some files, but not that one IIUC. The concern behind that rustbot check is to avoid conflicts on merges, but it's not a problem in this case and almost nobody follows up with a r-a PR anyway.